smpp/smpptest/server.go

```diff
@@ -71,11 +71,11 @@ func NewUnstartedServer(user, password string, port int) *Server {
 		User:    user,
 		Passwd:  password,
 		Handler: EchoHandler,
-		l:       newLocalListener(port),
+		l:       listener,
 	}
 }
 
-func newLocalListener(port int) net.Listener {
+func NewLocalListener(port int) net.Listener {
```

Comment:
If we're ever going to have a listener for tests, we better have the same semantics of the net package, which takes ip:port.